lib/web_ui/dev/safari_installation.dart

```diff
@@ -45,10 +49,75 @@ class SafariArgParser extends BrowserArgParser {
   String get version => _version;
 
   bool _isMobileBrowser;
-
   bool get isMobileBrowser => _isMobileBrowser;
 }
 
+class IosSafariArgParser extends BrowserArgParser {
+  static final IosSafariArgParser _singletonInstance = IosSafariArgParser._();
+
+  /// The [IosSafariArgParser] singleton.
+  static IosSafariArgParser get instance => _singletonInstance;
+
+  String get version => 'iOS ${iosMajorVersion}.${iosMinorVersion}';
+
+  int _pinnedIosMajorVersion;
+  int _iosMajorVersion;
+  int get iosMajorVersion => _iosMajorVersion ?? _pinnedIosMajorVersion;
+
+  int _pinnedIosMinorVersion;
+  int _iosMinorVersion;
+  int get iosMinorVersion => _iosMinorVersion ?? _pinnedIosMinorVersion;
+
+  String _pinnedIosDevice;
+  String _iosDevice;
+  String get iosDevice => _iosDevice ?? _pinnedIosDevice;
+
+  IosSafariArgParser._();
+
+  @override
+  void populateOptions(ArgParser argParser) {
+    final YamlMap browserLock = BrowserLock.instance.configuration;
+    _pinnedIosMajorVersion = browserLock['ios-safari']['majorVersion'] as int;
+    _pinnedIosMinorVersion = browserLock['ios-safari']['minorVersion'] as int;
+    _pinnedIosDevice = browserLock['ios-safari']['device'] as String;
+    argParser
+      ..addOption('majorVersion',
+          defaultsTo: '$_pinnedIosMajorVersion',
+          help: 'The major version for the iOS operating system the iOS '
+              'Simulator will use for tests. For example for testing with '
+              'iOS 13.2, use major version as 13. Use command: '
+              '`xcrun simctl list runtimes` to list available versions. Use '
+              'XCode to install more versions: XCode > Preferences > Components'
+              'If this value is not filled version locked in the '
+              'browser_lock.yaml file will be user.')
+      ..addOption('minorVersion',
```

Comment:
Why not just accept `--version=13.2`?

Reply:
> Do we need all the command-line options? Can we support using a custom simulator instance by picking whatever is currently running? Then, all you need to do to choose a different simulator is Cmd + Space, type "simulator" and select the one you want.

As discussed offline, a `booted` simulator does not necessarily means that it has to show on the Simulator UI. As agreed I'll keep the command line arguments.

Thanks for the suggestion.

Reply:
"Why not just accept --version=13.2" -> Changed the parameter. Thanks.